packages/plugin/src/Bundles/Backup/Export/FormieV3/FormProcessor.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Solspace\Freeform\Bundles\Backup\Export\FormieV3;
+
+use Solspace\Freeform\Bundles\Attributes\Property\PropertyProvider;
+use Solspace\Freeform\Bundles\Backup\Collections\FormCollection;
+use Solspace\Freeform\Bundles\Backup\Collections\PageCollection;
+use Solspace\Freeform\Bundles\Backup\DTO\Form;
+use Solspace\Freeform\Bundles\Backup\Export\FormieV3\Fields\FormieFieldMapper;
+use Solspace\Freeform\Bundles\Backup\Export\FormieV3\Layout\FormieLayoutBuilder;
+use Solspace\Freeform\Bundles\Backup\Export\FormieV3\Notifications\FormieNotificationProcessor;
+use Solspace\Freeform\Form\Settings\Implementations\ValueGenerators\RandomColorGenerator;
+use Solspace\Freeform\Form\Settings\Settings as FormSettings;
+use Solspace\Freeform\Freeform;
+use verbb\formie\elements\Form as FormieForm;
+
+class FormProcessor
+{
+    private FormieFieldMapper $fieldMapper;
+    private FormieLayoutBuilder $layoutBuilder;
+    private FormieNotificationProcessor $notificationProcessor;
+
+    public function __construct(private PropertyProvider $propertyProvider)
+    {
+        $this->fieldMapper = new FormieFieldMapper();
+        $this->layoutBuilder = new FormieLayoutBuilder($this->fieldMapper);
+        $this->notificationProcessor = new FormieNotificationProcessor();
+    }
```

Comment:
If we're using `Craft::$container->get()` to get the processor instance via the Dependency Injector, you can just specify all of these as dependencies directly.

```suggestion
    public function __construct(
        private PropertyProvider $propertyProvider,
        private FormieLayoutBuilder $layoutBuilder,
        private NotificationProcessor FormieNotificationProcessor,
    ) {
    }
```

The DI container will take care of instantiating everything and pass these itself